EntAppProject/src/main/resources/templates/JBLogin.html

```diff
@@ -1,45 +1,102 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
 <head>
-<title>Welcome to JobBook</title>
-<link rel="stylesheet" href="JobBookCSS.css">
+	<title>JobBook - Login</title>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+<!--===============================================================================================-->	
+	<link rel="icon" type="image/png" href="../static/images/icons/favicon.ico"/>
+<!--===============================================================================================-->
+	<link rel="stylesheet" type="text/css" href="../static/vendor/bootstrap/css/bootstrap.min.css">

```

Comment:
This is where the bootstrap file is located and referenced in the HTML page, if you want to replicate this in other files.